### DISCUSSION TITLE
Special edition merch

### DISCUSSION THREAD

**Discussion by benallfree** [pockethost/pockethost#404] [Polls]
For Founders and Flounders, we need to know what kind of commemorative merch you'd like. At signup it said tee shirt or mug, but we are open to other ideas.

**Comment by gwashark** [comment]
Vote plushie. I want to see what will come out!

**Comment by humbertomorais** [comment]
I subscribed to the Starter Plan and it was not updated on the Dashboard. My email is [humberto.silva@ifma.edu.br](mailto:humberto.silva@ifma.edu.br). Could you help me?
